### PR TITLE
feat: New doc add

### PR DESCRIPTION
**Pull request by Joshyahweh**

### CODE REVIEW

File: app/upload/[id]/page.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @next/next/no-img-element */
```
Comment (teebarg) [inline app/upload/[id]/page.js]: remove this, use Next Image or disable it on the appropriate line(ref https://nextjs.org/docs/pages/api-reference/components/image)

File: components/User.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @next/next/no-img-element */
```
Comment (teebarg) [inline components/User.js]: remove this, see comment above

File: components/User.js [hunk 3]
```diff
@@ -6,7 +7,7 @@ const myArray = [1, 2, 3, 4];
 
 const imgSrc = (img) => {
    if (!img) {
-      return "@/public/images/avatar.webp";
+      return "@public/images/avatar.webp";
```
Comment (teebarg) [inline components/User.js]: what's this change for?

**Comment by teebarg** [comment]
Remove package-lock.json from this PR

File: components/User.js (hunk 3 above)
Comment (Joshyahweh) [reply components/User.js]: To use the @ component path.

Comment (teebarg) [reply components/User.js]: Why remove the forward slash?

**Comment by Joshyahweh** [comment]
> Remove package-lock.json from this PR

package-lock.json is not on this PR

File: components/User.js (hunk 3 above)
Comment (Joshyahweh) [reply components/User.js]: I will add it back sir

**Review by teebarg — APPROVED** [review]

**Pull request merged**